sentence.go

```diff
@@ -19,6 +19,66 @@ const (
 	ChecksumSep = "*"
 )
 
+var (
+	customParsers          = map[string]ParserCallback{}
```

Comment:
Protect access to `customParsers` with a `sync.Mutex`. Call it `customParsersMu`.

Reply:
ok